src/scimlfunctions.jl

```diff
@@ -2102,21 +2088,13 @@ struct OptimizationFunction{iip, AD, F, G, H, HV, C, CJ, CH, LH, HP, CJP, CHP, L
     cons::C
     cons_j::CJ
     cons_h::CH
-    lag_h::LH
     hess_prototype::HP
     cons_jac_prototype::CJP
     cons_hess_prototype::CHP
-    lag_hess_prototype::LHP
     syms::S
     paramsyms::S2
-    observed::O
```

Comment:
? where did this go?

Reply:
It was a mistake, I forgot that these are used in MTK